From 4dccedb596d0db79e167965474cb579a94488e62 Mon Sep 17 00:00:00 2001
From: Dmytro Kelzikov <d.kelzikov@inango-systems.com>
Date: Fri, 6 Aug 2021 15:45:18 +0300
Subject: [PATCH] Commit Title          : Integrate CH. OVS daemons are
 restarted in case they freeze or killed

BugID                 : 25222
Bug Fixed(y/N[blankN]): N
CodeReviewId          : 0
Description           : Replaced the Restart option from on-failure to always in ovs-vswitchd and ovsdb services.
---
 rhel/usr_lib_systemd_system_ovs-vswitchd.service.in | 2 +-
 rhel/usr_lib_systemd_system_ovsdb-server.service    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/rhel/usr_lib_systemd_system_ovs-vswitchd.service.in b/rhel/usr_lib_systemd_system_ovs-vswitchd.service.in
index a2977046b..13e3bfeb3 100644
--- a/rhel/usr_lib_systemd_system_ovs-vswitchd.service.in
+++ b/rhel/usr_lib_systemd_system_ovs-vswitchd.service.in
@@ -10,7 +10,7 @@ PartOf=openvswitch.service
 [Service]
 Type=forking
 PIDFile=/var/run/openvswitch/ovs-vswitchd.pid
-Restart=on-failure
+Restart=always
 Environment=XDG_RUNTIME_DIR=/var/run/openvswitch
 EnvironmentFile=/etc/openvswitch/default.conf
 EnvironmentFile=-/nvram/openvswitch.conf
diff --git a/rhel/usr_lib_systemd_system_ovsdb-server.service b/rhel/usr_lib_systemd_system_ovsdb-server.service
index a23154515..5ee803ec0 100644
--- a/rhel/usr_lib_systemd_system_ovsdb-server.service
+++ b/rhel/usr_lib_systemd_system_ovsdb-server.service
@@ -9,7 +9,7 @@ PartOf=openvswitch.service
 [Service]
 Type=forking
 PIDFile=/var/run/openvswitch/ovsdb-server.pid
-Restart=on-failure
+Restart=always
 EnvironmentFile=/etc/openvswitch/default.conf
 EnvironmentFile=-/nvram/openvswitch.conf
 ExecStartPre=/usr/bin/chown ${OVS_USER_ID} /var/run/openvswitch /var/log/openvswitch
